tests/resources/rebuild_example_data

```diff
@@ -112,8 +91,6 @@ def main(ctx, rebuild_input):
         # The environ here should be the same (modulo the tempdir location)
         # as we use in test_examples.py.
         environ = {'ZIPLINE_ROOT': d.getpath('example_data/root')}
-        if rebuild_input:
-            rebuild_input_data(environ)
```

Comment:
Do the tests no longer rely on this input data? Or how should we rebuild it from now on?